docs/diagrams/AddScoreToModelSequenceDiagram.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+!include style.puml
+
+mainframe **sd** add score to model\t
+
+box Logic LOGIC_COLOR_T1
+participant "addScoreCommand\n:AddScoreCommand" as AddScoreCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "assessmentToEdit\n:Assessment" MODEL_COLOR
+end box
+
+AddScoreCommand -> Model : getAssessmentList()
+AddScoreCommand -> Model : getStudentList()
+
+AddScoreCommand -> AddScoreCommand : getToEditAssessment()
```

Comment:
Perhaps it's a good idea to include the arguments in here?
Right now it feels like you're calling getToEditAssessment method of the AssessmentList. (At least that's what I thought when looking at this)

Reply:
Hm I didn't include the arguments because `scoreDescriptor` is part of the argument list, but the `scoreDescriptor` passed into `getToEditAssessment()` is not the same instance as the `scoreDescriptor` shown in the main sequence diagram, thus I'm afraid putting it in might cause confusion ;-; What do you think?

Argument list of `getToEditAssessment()`
![image](https://user-images.githubusercontent.com/63232300/140635553-055c3408-ce04-4fa1-b084-e4d4b7d7bdb6.png)

Different instance of `scoreDescriptor`
![image](https://user-images.githubusercontent.com/63232300/140635592-ef94ce38-6237-4225-9241-b0477c8f709e.png)

Reply:
Oh. I can definitely see that causing some problems. This way is fine then!